.github/pull_request_template.md

```diff
@@ -34,14 +34,20 @@ Put an `x` in the boxes that apply. You can also fill these out after creating t
 them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
 -->
 
-- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
-- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
+- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
```

Comment:
Done, to potentially save you some headache in the future: in this repo it's actually `git checkout origin/main ...`... That took me some time to get. ^^

Reply:
That's only the case if your local `master` branch is not in sync with its remote counterpart at Github (origin/master). To prevent such problems in the future, please install https://www.git-town.com and run `git sync` regularly (like every hour or so).

Reply:
no, I always do `git fetch` regularly. :) I meant the `main` instead of `master` part.

Reply:
Oh right, this repo uses `main`! Also, you are working off a fork.  In this case, please create an `upstream` remote pointing to the original repo:

```
git remote add upstream git@github.com:ory/elements.git
```

Now, when you run `git sync`, it does the right things:

- it fetches from `upstream/main`
- it updates your local branches (main and your feature branch)
- it pushes these updates to your fork an Github

You end up with a setup where all branches in your fork are in sync with the forked repo, and you can therefore use your local branches for all work. No need to ever access remotes in Git commands.

Also, now that your branch is shipped, please run `git prune-branches` to let Git Town remove the now obsolete feature branch your local Git workspace.